src/browser/HostInstaller.cpp

```diff
@@ -0,0 +1,233 @@
+/*
+*  Copyright (C) 2017 Sami Vänttinen <sami.vanttinen@protonmail.com>
+*
+*  This program is free software: you can redistribute it and/or modify
+*  it under the terms of the GNU General Public License as published by
+*  the Free Software Foundation, either version 3 of the License, or
+*  (at your option) any later version.
+*
+*  This program is distributed in the hope that it will be useful,
+*  but WITHOUT ANY WARRANTY; without even the implied warranty of
+*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*  GNU General Public License for more details.
+*
+*  You should have received a copy of the GNU General Public License
+*  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "HostInstaller.h"
+#include <QDir>
+#include <QFile>
+#include <QStandardPaths>
+#include <QJsonArray>
+#include <QJsonDocument>
+#include <QCoreApplication>
+#include <QMessageBox>
+
+const QString HostInstaller::HOST_NAME = "com.varjolintu.keepassxc_browser";
+const QStringList HostInstaller::ALLOWED_ORIGINS = QStringList()
+    << "chrome-extension://iopaggbpplllidnfmcghoonnokmjoicf/"
+    << "chrome-extension://fhakpkpdnjecjfceboihdjpfmgajebii/"
+    << "chrome-extension://jaikbblhommnkeialomogohhdlndpfbi/";
+
+const QStringList HostInstaller::ALLOWED_EXTENSIONS = QStringList()
+    << "keepassxc-browser@sami.vanttinen";
```

Comment:
To change this I probably need to delete the extension from AMO and create a new one. UUID cannot be changed afterwards.

Reply:
Perhaps not a big issue. This can be done when we make the transition from an unofficial patch to an official feature. If we do that, we could perhaps re-upload it with an official KeePassXC account.

Reply:
Yes. This should be done before the official release.

Reply:
You should change this now, before we merge.